src/zenml/integrations/gcp/image_builders/gcp_image_builder.py

```diff
@@ -72,14 +72,11 @@ def validator(self) -> Optional["StackValidator"]:
         def _validate_remote_components(stack: "Stack") -> Tuple[bool, str]:
             assert stack.container_registry
 
-            if (
-                stack.container_registry.flavor
-                != ContainerRegistryFlavor.GCP.value
-            ):
+            if stack.container_registry.config.is_local:
```

Comment:
@htahir1 I changed this to require a remote container registry now, which is sensible I think.

There is another check for the artifact store flavor below, which also just allows GCP ones. Is that something that you want removed as well?

Reply:
Is the default container registry also remote? For artifact stores id also remove that tbh

Reply:
A container registry only counts as local if it's a localhost URL.

As for the artifact store: We have similar checks in other components (e.g. s3 artifact store in AWSImageBuilder). How far do we go with removing them?

Reply:
Do whats easiest and logical for you @schustmi , the skip validation env variable is always there